tests/integration/aliases_test/test_aliases.py

```diff
@@ -1,70 +0,0 @@
-from tests.integration.base import DBTIntegrationTest, use_profile
-
-
-class TestAliases(DBTIntegrationTest):
-    @property
-    def schema(self):
-        return "aliases"
-
-    @property
-    def models(self):
-        return "models"
-
-    @property
-    def project_config(self):
-        return {
-            'config-version': 2,
-            "macro-paths": ['macros'],
-            "models": {
-                "test": {
-                    "alias_in_project": {
-                        "alias": 'project_alias',
-                    },
-                    "alias_in_project_with_override": {
-                        "alias": 'project_alias',
-                    },
-                }
-            }
-        }
-
-    @use_profile('bigquery')
-    def test__alias_model_name_bigquery(self):
-        results = self.run_dbt(['run'])
-        self.assertEqual(len(results), 4)
-        self.run_dbt(['test'])
-
-
-class TestSameAliasDifferentDatabases(DBTIntegrationTest):
-    setup_alternate_db = True
-
-    @property
-    def schema(self):
-        return "aliases_026"
-
-    @property
-    def models(self):
-        return "models-dupe-custom-database"
-
-    @property
-    def project_config(self):
-        return {
-            'config-version': 2,
-            "macro-paths": ['macros'],
-            'models': {
-                'test': {
-                    'alias': 'duped_alias',
-                    'model_b': {
-                        'database': self.alternative_database,
-                    },
-                },
-            }
-        }
```

Comment:
would like to verify which class/test we are really wanting to do here, noticing this one in the current conversion is leaving two or one model left in bigquery every run not exactly sure why off top of my head will continue to look into it. would love to hear any ideas though?? 

ref to new test [TestSameTestSameAliasDifferentSchemasBigQuer](https://github.com/dbt-labs/dbt-bigquery/pull/318/files#diff-ca3edbc2e0b6572e21f4fcf04747fb68718878bdf9eee5d93efb079c2a08cf19)